### DISCUSSION TITLE
has already been used to obtain an academic discount on GitHub account(s)

### DISCUSSION THREAD

**Discussion by 4RKO0** [community/community#138123] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I unlinked my school email and my own email that was linked to my previous github account and linked it to a new account (because I didn't remember the password of my previous account) and I wanted to use copilot on my new account I tried to get the copilot package for students like I did on my other account but I got the attached error. “Already used to get academic discount on GitHub account(s)”

![github_copilot](https://github.com/user-attachments/assets/9b14fb2f-3836-4af9-9166-70fed90dc321)

**Comment by VadymMakohon** [comment]
It seems like GitHub recognizes that your school email has already been used to claim the academic discount, likely on your previous account. Here are a few suggestions to resolve the issue:

Recover Your Previous GitHub Account:
Try recovering access to your previous GitHub account by using the "Forgot Password" option and resetting it via your linked email. Once you regain access, you can continue using the student benefits, including Copilot.

Contact GitHub Support:
If you can't recover the old account, reach out to [GitHub Support](https://support.github.com/contact). Explain the situation—mention that you've unlinked your school email from the old account and now want to link it to the new one to use Copilot for students. They might be able to help transfer your student benefits.

Use a Different Email for Verification:
If you have another school-issued email address, you could try using that to apply for the student package on your new account.

Close the Previous Account:
If you no longer need your old account, you could try requesting GitHub Support to delete the account associated with the academic discount. Once it's removed, you should be able to apply the benefits to your new account.

I hope this helps.
